### ISSUE TITLE
whipper doesn't name files .flac, which leads to it not being able to find ripped files

### ISSUE THREAD

**Issue by Freso** [JoeLametta/whipper#194]
Using the command `whipper cd --device='/dev/cdrom' rip --cdr --prompt --working-directory='/tmp/rip-disc.dz2' --output-directory='' --track-template='%A - %d (%y) [%X]/%t. %a - %n' --disc-template='%A - %d (%y) [%X]/%A - %d' --release-id=''` to rip a disc, the disc seems to rip 100% perfectly fine, yet at the end (AccurateRip verification?), it bails out: 

```Python traceback
Traceback (most recent call last):
  File "/usr/bin/whipper", line 11, in <module>
    load_entry_point('whipper==0.5.1', 'console_scripts', 'whipper')()
  File "/usr/lib/python2.7/site-packages/whipper/command/main.py", line 32, in main
    ret = cmd.do()
  File "/usr/lib/python2.7/site-packages/whipper/command/basecommand.py", line 124, in do
    return self.cmd.do()
  File "/usr/lib/python2.7/site-packages/whipper/command/basecommand.py", line 124, in do
    return self.cmd.do()
  File "/usr/lib/python2.7/site-packages/whipper/command/cd.py", line 190, in do
    self.doCommand()
  File "/usr/lib/python2.7/site-packages/whipper/command/cd.py", line 488, in doCommand
    self.program.verifyImage(self.runner, self.ittoc)
  File "/usr/lib/python2.7/site-packages/whipper/common/program.py", line 573, in verifyImage
    verifytask = image.ImageVerifyTask(cueImage)
  File "/usr/lib/python2.7/site-packages/whipper/image/image.py", line 146, in __init__
    path = image.getRealPath(index.path)
  File "/usr/lib/python2.7/site-packages/whipper/image/image.py", line 67, in getRealPath
    return self.cue.getRealPath(path)
  File "/usr/lib/python2.7/site-packages/whipper/image/cue.py", line 187, in getRealPath
    return common.getRealPath(self._path, path)
  File "/usr/lib/python2.7/site-packages/whipper/common/common.py", line 240, in getRealPath
    raise KeyError("Cannot find file for %r" % filePath)
KeyError: "Cannot find file for u'01. Sofia Jannok - \\xc1\\u0161\\u0161og\\xe1ttis'"
```
```text
> ls -Al /tmp/rip-disc.dz2/*
totalt 189236
-rw------- 1 freso freso 19920750 19 sep 15:22 '01. Sofia Jannok - Áššogáttis'
-rw------- 1 freso freso 18421679 19 sep 15:24 '02. Sofia Jannok - Árvas'
-rw------- 1 freso freso 19433438 19 sep 15:25 '03. Sofia Jannok - Liekkas'
-rw------- 1 freso freso 22212062 19 sep 15:26 '04. Sofia Jannok - Irene'
-rw------- 1 freso freso 18019945 19 sep 15:28 '05. Sofia Jannok - Davádat'
-rw------- 1 freso freso 18746093 19 sep 15:29 '06. Sofia Jannok - Váralaš'
-rw------- 1 freso freso 23147567 19 sep 15:30 '07. Sofia Jannok - Sámi eatnan duoddariid'
-rw------- 1 freso freso  2414696 19 sep 15:31 '08. Sofia Jannok - Vuolvojávrri'
-rw------- 1 freso freso 15998419 19 sep 15:31 '09. Sofia Jannok - Golle Máze'
-rw------- 1 freso freso 18971715 19 sep 15:33 '10. Sofia Jannok - Vássán áiggit'
-rw------- 1 freso freso 16464373 19 sep 15:34 '11. Sofia Jannok - Dulla lulla'
-rw-r--r-- 1 freso freso     1326 19 sep 15:34 'Sofia Jannok - Áššogáttis.cue'
-rw-r--r-- 1 freso freso      825 19 sep 15:34 'Sofia Jannok - Áššogáttis.m3u'
> yaourt -Qi whipper-git|grep -i version
Version                : 0.5.1.r68.ga3e9260-1
```

**Comment by RecursiveForest** [comment]
Right now you need to use `%x` in the track template to add the track extension, but it's definitely a bug that whipper doesn't detect these files.

Kind fascinating how poorly orthogonalised morituri was… it's been hard to make refactors because of how difficult integration testing is right now.

**Comment by Freso** [comment]
Adding `%x` (or using the patch in #195) will just result in https://github.com/JoeLametta/whipper/issues/196 though. :)

**Comment by xmixahlx** [comment]
Just confirming that adding an .%x to the end of the track template config solves this problem for me. I did not encounter #196.

**Comment by JoeLametta** [comment]
Solved by RecursiveForest's PR #195.
